views/partials/post/post-cards.blade.php

```diff
@@ -15,9 +15,9 @@
                    @slot('subHeading')
                    @typography(['variant' => 'meta', 'element' => 'p'])
                         @icon(['icon' => 'date_range']) @endicon
-                        @date(['action' => 'formatDate', 'format' => 'D d M Y H:i' ,'timestamp' => $post->startDate])@enddate
+                        {{ \EventManagerIntegration\App::formatEventDate($post->startDate, $post->endDate) }}
```

Comment:
Wrap this in @date component but leave action blank: 

@date(['action' => '', 'timestamp' => \EventManagerIntegration\App::formatEventDate($post->startDate, $post->endDate)])@enddate

This will output correct html wrapper.

Reply:
Fixed 👍